sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -314,11 +314,17 @@ private[sql] abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
 
   object DDLStrategy extends Strategy {
     def apply(plan: LogicalPlan): Seq[SparkPlan] = plan match {
-      case CreateTableUsing(tableName, userSpecifiedSchema, provider, true, options) =>
+      case CreateTableUsing(tableName, userSpecifiedSchema, provider, true, opts, allowExisting) =>
         ExecutedCommand(
-          CreateTempTableUsing(tableName, userSpecifiedSchema, provider, options)) :: Nil
+          CreateTempTableUsing(
+            tableName, userSpecifiedSchema, provider, opts, allowExisting)) :: Nil
+      case CreateTableUsing(_, _, _, false, _, _) =>
```

Comment:
Maybe `case c: CreateTableUsing if !c.temporary =>` for clarity and readability here.